odxtools/odxlink.py

```diff
@@ -60,6 +60,9 @@ def __eq__(self, other: Any) -> bool:
         # i.e. the same OdxId object can be put into all of them.
         return self.local_id == other.local_id
 
+    def __str__(self):
+        return f"OdxLinkId('{self.local_id}')"
```

Comment:
include doc_name if present

Reply:
we only have a list of document fragments which do not necessarily correspond to the file name of the XML document in the zip file. The idea here is to give people enough to `grep` for the ID in an unpacked PDX while getting a unique result for ~99% of all cases and at the same time not cluttering up the output too much...